plugins/foreman_remote_execution/0.0/index.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: plugin
+title: Foreman Remote Execution 0.0 Manual
+version: 0.0
```

Comment:
The released version is 0.0.1 and the manual will reflect only minor versions, not revisions, hence 0.0. Hopefully people will understand this is very early version. With frequent release cadence we should get to 0.1 in few weeks.